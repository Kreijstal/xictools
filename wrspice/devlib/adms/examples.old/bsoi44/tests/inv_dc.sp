 Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
* Modified by Darsen Lu 03/11/2009
* Modified by Tanvir Morshed 11/21/2010

.include ./nmos4p4.mod
.include ./pmos4p4.mod
.option TEMP=27C

Vpower VD 0 1.5
Vgnd VS 0 0
Vgate Gate 0 0.0
MN0 VS Gate Out VS N1 W=10u L=0.18u
MP0 VD Gate Out VS P1 W=20u L=0.18u

.dc Vgate 0 1.5 0.05
.print dc v(out)

.control
run
plot v(Out)
.endc

.END
