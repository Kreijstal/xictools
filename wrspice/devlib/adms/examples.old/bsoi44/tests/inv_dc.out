
Circuit:  Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Circuit:  Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Date: Thu Dec 16 23:37:44  2010


 Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
DC transfer characteristic  Thu Dec 16 23:37:44  2010
--------------------------------------------------------------------------------
Index   sweep           v(out)          
--------------------------------------------------------------------------------
0	0.000000e+00	1.499923e+00	
1	5.000000e-02	1.499949e+00	
2	1.000000e-01	1.499951e+00	
3	1.500000e-01	1.499845e+00	
4	2.000000e-01	1.499094e+00	
5	2.500000e-01	1.495893e+00	
6	3.000000e-01	1.488442e+00	
7	3.500000e-01	1.476665e+00	
8	4.000000e-01	1.460776e+00	
9	4.500000e-01	1.440492e+00	
10	5.000000e-01	1.415038e+00	
11	5.500000e-01	1.382649e+00	
12	6.000000e-01	1.339928e+00	
13	6.500000e-01	1.279077e+00	
14	7.000000e-01	1.176847e+00	
15	7.500000e-01	8.868719e-01	
16	8.000000e-01	2.308297e-01	
17	8.500000e-01	1.295976e-01	
18	9.000000e-01	8.482177e-02	
19	9.500000e-01	5.850477e-02	
20	1.000000e+00	4.072133e-02	
21	1.050000e+00	2.779857e-02	
22	1.100000e+00	1.820482e-02	
23	1.150000e+00	1.121928e-02	
24	1.200000e+00	6.359812e-03	
25	1.250000e+00	3.253097e-03	
26	1.300000e+00	1.486166e-03	
27	1.350000e+00	6.086026e-04	
28	1.400000e+00	2.277695e-04	
29	1.450000e+00	8.592251e-05	

CPU time since last call: 0.000 seconds.

Total CPU time: 0.000 seconds.

Current dynamic memory usage = 266240,
Dynamic memory limit = -9547777.

